### PR TITLE
Basic CouchDB integration

### PR DESCRIPTION
**Pull request by scull7**

### CODE REVIEW

File: lib/couchdb/error.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const R = require('ramda')
+const Response = require('./response.js')
+
+
+const OBJECT     = 'Object'
+const PROP_ERROR = 'error'
+const DEFAULT_MESSAGE = 'Generic CouchDB Error'
+
+
+const hasError = R.both( R.is(OBJECT), R.has(PROP_ERROR))
```
Comment (jxbadam) [inline lib/couchdb/error.js]: Spacing here after the paren is off :(

File: lib/couchdb/index.js [hunk 2]
```diff
@@ -1,5 +1,111 @@
 
-const GotCouch = require('got-couch')
+const R            = require('ramda')
+const Bluebird     = require('bluebird')
+const CouchWrapper = require('./wrapper.js')
+const ID           = require('../identifier.js')
+const Err          = require('../error')
```
Comment (jxbadam) [inline lib/couchdb/index.js]: Why no .js ... is this a directory include?

File: lib/couchdb/wrapper.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
+
+
```
Comment (jxbadam) [inline lib/couchdb/wrapper.js]: Blank spaces to begin files?

File: test/lib/couchdb/index.spec.js [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+  , side: 'villain'
+  }
+, { username: 'anarky'
+  , name_first: 'Lonnie'
+  , name_last: 'Machin'
+  , side: 'both'
+  }
+, { username: 'constantine'
+  , name_first: 'John'
+  , name_last: 'constantine'
```
Comment (jxbadam) [inline test/lib/couchdb/index.spec.js]: Lower case last name... this man is no hero.

**Review by ricejasonf — APPROVED** [review]
:shipit:

**Comment by scull7** [comment]
@jxbadam fixed

**Comment by jxbadam** [comment]
I see red x's :(

**Pull request merged**